pkg/fanal/analyzer/analyzer.go

```diff
@@ -56,7 +56,14 @@ type SecretScannerOption struct {
 
 type LicenseScannerOption struct {
 	// Use license classifier to get better results though the classification is expensive.
-	Full                      bool
+	// Flag indicates whether license scanner is enabled or not
+	Enabled bool
```

Comment:
Why cant we use the `Full` scan bool here?

Reply:
Full corresponds to --license-full flag
Enabled -> --scanners license

So just to validate properly, only when both the flags are passed, deep license scan is enabled for us.

Reply:
But how are both different? --license-full vs --scanners